cloud/edgecontroller/pkg/controller/manager/location.go

```diff
@@ -109,6 +114,26 @@ func (lc *LocationCache) SecretNodes(namespace, name string) (nodes []string) {
 	return
 }
 
+//GetNodeRole returns role for the given node name
+func (lc *LocationCache) GetNodeRole(nodeName string) string {
```

Comment:
What are the valid returned values?

Reply:
I don't think Kubernetes has a restriction on the value of role since it is a label.